src/Sfa.Tl.Matching.Data/Repositories/OpportunityRepository.cs

```diff
@@ -223,25 +225,49 @@ public int GetEmployerOpportunityCount(int opportunityId)
                 item.OpportunityId == opportunityId && item.IsSaved && !item.IsCompleted);
         }
 
+        public async Task<List<MatchingServiceOpportunityReportDto>> GetMatchingServiceOpportunityReportAsync()
```

Comment:
Rename this to GetPowerBIOpportunityReportAsync and GetPowerBIProviderOpportunityReportAsync to High Light PowerBI Dependency